util/container/Dockerfile

```diff
@@ -38,6 +38,8 @@ RUN wget https://github.com/llvm/circt/releases/download/firtool-1.42.0/firrtl-b
     rm firrtl-bin-ubuntu-20.04.tar.gz
 
 # Install generic LLVM toolchain in /usr/bin - latest stable release is 17
-RUN  wget https://apt.llvm.org/llvm.sh && chmod +x llvm.sh && ./llvm.sh 17 && rm llvm.sh
+RUN  wget https://apt.llvm.org/llvm.sh && chmod +x llvm.sh && ./llvm.sh 17 && rm llvm.sh && \
+sed -i 's/AR = ar/AR = llvm-ar-17/g; s/CXX = g++/CXX = clang++-17/g; s/LINK = g++/LINK = ld.lld-17/g' /usr/share/verilator/include/verilated.mk
```

Comment:
```suggestion
sed -i 's/AR = ar/AR = llvm-ar-17/g; s/CXX = g++/CXX = clang++-17/g; s/LINK = g++/LINK = ld.lld-17/g' /verilator/include/verilated.mk
```
This I think?

Reply:
seems like verilater is just installed in the root folder

Reply:
Yes I fixed this haha but there's a different problem like it does not seem to be compatible with cocotb ^^;